openapi/servicenow/client.bal

```diff
@@ -0,0 +1,477 @@
+        map<anydata> queryParam = {"sysparm_dependent_value": sysparmDependentValue, "sysparm_limit": sysparmLimit, "sysparm_offset": sysparmOffset, "sysparm_reference_field_columns": sysparmReferenceFieldColumns, "sysparm_query": sysparmQuery, "sysparm_ref_qual_input": sysparmRefQualInput};
+        path = path + check getPathForQueryParam(queryParam);
+        map<any> headerValues = {"Accept": accept};
+        map<string|string[]> accHeaders = getMapForHeaders(headerValues);
+        json response = check self.clientEp-> get(path, accHeaders, targetType = json);
+        return response;
+    }
+    # Retrieves the specified Customer Service Management (CSM) case.
+    #
+    # + id - Sys_id or case number for the case to retrieve. Located in the Case [sn_customerservice_case] table. Results will be unpredictable if a specified case number is shared by two or more cases.
+    # + sysparmDisplayValue - Data retrieval operation for reference and choice fields. Based on this value, retrieves the display value and/or the actual value from the database.
+    # + accept - Data format of the response body.
+    # + return - If successful, returns a JSON object. Otherwise returns the relevant error.
+    remote isolated function getCSMCaseById(string id, string? sysparmDisplayValue = (), string? accept = ()) returns json|error {
+        string  path = string `/api/sn_customerservice/case/${id}`;
+        map<anydata> queryParam = {"sysparm_display_value": sysparmDisplayValue};
+        path = path + check getPathForQueryParam(queryParam);
+        map<any> headerValues = {"Accept": accept};
+        map<string|string[]> accHeaders = getMapForHeaders(headerValues);
+        json response = check self.clientEp-> get(path, accHeaders, targetType = json);
+        return response;
+    }
+    # Updates the specified existing Customer Service Management (CSM) case with the passed-in parameters.
+    #
+    # + id - Sys_id of the case to update. Located in the Case [sn_customerservice_case] table.
+    # + payload - Field name and the associated value for each parameter to define in the specified record in JSON format.
+    # + sysparmDisplayValue - Data retrieval operation for reference and choice fields. Based on this value, retrieves the display value and/or the actual value from the database.
+    # + sysparmFields - Comma-separated list of fields to return in the response.
+    # + sysparmGetCaseDetails - Flag that indicates whether to return the updated case record.
+    # + accept - Data format of the response body.
+    # + contentType - Data format of the request body.
+    # + return - If successful, returns a JSON object. Otherwise returns the relevant error.
+    remote isolated function updateCSMCaseById(string id, json payload, string? sysparmDisplayValue = (), string? sysparmFields = (), string? sysparmGetCaseDetails = (), string? accept = (), string? contentType = ()) returns json|error {
```

Comment:
The function name must be `updateCSMCase`

Reply:
Fixed by https://github.com/kasthuriraajan/ballerinax-openapi-connectors/commit/7d722e1656dd4084768a6a6f0aab1753b97a5d98